### DISCUSSION TITLE
Redirection not working well for my domain.

### DISCUSSION THREAD

**Discussion by smart8099** [danirus/django-comments-xtd#424] [Q&A] [answered]
Hello guys, 
Am using django_comments_xtd library. I followed the documentation and I was able to implement it in my local environment. It worked perfectly in my local environment. It uses the SITE_ID parameter in the settings file and use the record in the django_site table. In my production environment. I created record for  the django site, it is pointed to my domain (let's say, example.com). I have redirection issues when a comment is made.

it redirects to [https://https//example.com/buster/reports/ice-cream/#c4](https://https//fakebusters.com/buster/early-sensor-reports/ice-cream/#c4) instead of 
[https://example.com/buster/reports/ice-cream/#c4](https://https//fakebusters.com/buster/early-sensor-reports/ice-cream/#c4)

In my production environment the record for the SITE_ID points to example.com and  not https://example.com. I tried https://example.com in the table, and I was still getting same error.

What might be the cause of that. 

Note: I followed the entire documentation and I didn't see a solution to that. 
Thanks.

**Comment by danirus** [comment, marked as answer]
Hi @smart8099, what is the content of your site object in the DB?

    from django.contrib.sites.models import Site
    print(Site.objects.get(pk=1).domain)

Check if you added the `protocol` to the domain, maybe that is the problem.

**Reply by smart8099** [reply]
Hello @danirus ,

Thanks for the reply. when i print the Site.objects.get(pk=1).domain, I get my domain (example.com). No protocol was added.

**Comment by danirus** [comment]
And what about the implementation of `get_absolute_url` in the model that will receive comments?

**Reply by smart8099** [reply]
Yes, I implemented get_absolute_url in the model that will receive comment. And it worked perfectly in my local environment. it is now in the production environment

**Reply by danirus** [reply]
Does it happen using the javascript plugin? Can I see the code of your Django project?

**Reply by smart8099** [reply]
Thanks @danirus . I was able to resolve the issue. After changing the url in the django_site table, I had to restart my django application for that to take effect. Everything is working fine now. Thanks for the  support once again!
